### PR TITLE
JA: add site/ja/tutorials/text/text_classification_rnn.ipynb

### PR DESCRIPTION
**Pull request by masa-ita**

### CODE REVIEW

**Comment by AseiSugiyama** [comment]
@masa-ita This PR contains the operation to remove `site/ja/tutorials/text/word_embeddings.ipynb` (https://github.com/tensorflow/docs/pull/1174/commits/94a4ed58cc15a61b5a5756ef230e4baa143fc537) and it causes the conflict.  I guess it is better to make the notebook leave. Could you fix it?

File: site/ja/tutorials/text/text_classification_rnn.ipynb [hunk 1]
```diff
@@ -0,0 +1,743 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "bgs6nnSTGw-t"
+   },
+   "source": [
+    "`tf.keras.Sequential` モデルを構築し、最初に Embedding レイヤーから始めます。Embedding レイヤーは単語一つに対して一つのベクトルを収容します。呼び出しを受けると、Embedding レイヤーは単語のインデックスのシーケンスを、ベクトルのシーケンスに変換します。これらのベクトルは訓練可能です。（十分なデータで）訓練されたあとは、おなじような意味をもつ単語は、しばしばおなじようなベクトルになります。\n",
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/tutorials/text/text_classification_rnn.ipynb]: Thanks. I'll fix it.

File: site/ja/tutorials/text/text_classification_rnn.ipynb [hunk 2]
```diff
@@ -0,0 +1,743 @@
...
+    "\n",
+    "このインデックス参照は、ワンホットベクトルを `tf.keras.layers.Dense` レイヤーを使って行うおなじような演算に比べてずっと効率的です。\n",
+    "\n",
+    "リカレントニューラルネットワーク（RNN）は、シーケンスの入力を要素を一つずつ扱うことで処理します。RNN は、入力の一つのタイムステップからの出力を次のタイムステップに渡します。\n",
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/tutorials/text/text_classification_rnn.ipynb]: Thanks. It's better.

File: site/ja/tutorials/text/text_classification_rnn.ipynb [hunk 3]
```diff
@@ -0,0 +1,743 @@
...
+    "\n",
+    "RNN レイヤーとともに、`tf.keras.layers.Bidirectional` ラッパーを使用することができます。このラッパーは、入力を RNN 層の順方向と逆方向に伝え、その後出力を結合します。これにより、RNN は長いレンジの依存関係を学習することができます。"
```
[the remark this answers is not in the document]
Comment (masa-ita) [reply site/ja/tutorials/text/text_classification_rnn.ipynb]: Thank you.  It's natural.

**Comment by masa-ita** [comment]
@AseiSugiyama 
Thank you for your review.
I made a mistake to rebase this branch to remove the commit that cause a conflict. 
I'd close this PR and create a new one.

**Pull request closed without merging**